plugins/browser-plugin-form-tracking/src/helpers.ts

```diff
@@ -160,14 +161,30 @@ export function addFormListeners(tracker: BrowserTracker, configuration: FormTra
  */
 function getConfigurationForOptions(options?: FormTrackingOptions) {
   if (options) {
+    var formFilter = (_: HTMLElement) => true;
+    var forms: HTMLCollectionOf<HTMLFormElement> | null = null;
+    if (
+      !options.forms ||
+      (options.forms as any).allowlist ||
+      (options.forms as any).denylist ||
+      (options.forms as any).filter
```

Comment:
I don't love this solution to identify a `FilterCriterion` by duck typing, using the properties it might have (might lead to bugs in the future in case we add a property to the `FilterCriterion` type but forget to check for them here). But I can't think of a better solution, let me know if you know of one.

Reply:
The only thing I can think of is doing something like [this](https://www.npmjs.com/package/ts-transformer-keys), but adding in a package just for this seems a bit overkill (doubly so since it involves messing with the build). Perhaps add in a comment on the `FormTrackingOptions` interface and here as a reminder to update both? I think it's going to be less than ideal either way.

Reply:
Perhaps these `any`'s could be changed to `FilterCriterion<HTMLElement>` to be a bit more explicit?

Reply:
Thanks Greg, I added the type cast to `FilterCriterion<HTMLElement>`